### PR TITLE
Feature/liayoo/update bandwidth budget params

### PR DESCRIPTION
**Pull request by liayoo**
Updated `bandwidth_budget_per_block`, `service_bandwidth_budget_ratio`, `apps_bandwidth_budget_ratio` and `free_bandwidth_budget_ratio` blockchain params with bandage files and timer flags.

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/927

### CODE REVIEW

File: blockchain-configs/mainnet-prod/timer_flags.json [hunk 1]
```diff
@@ -22,5 +22,9 @@
   "update_max_state_tree_size_per_byte2": {
     "enabled_block": 58300,
     "has_bandage": true
+  },
+  "update_bandwidth_budget_params": {
+    "enabled_block": -1,
```
Comment (platfowner) [inline blockchain-configs/mainnet-prod/timer_flags.json:27]: could you use 'null' instead?

File: blockchain-configs/testnet-prod/timer_flags.json [hunk 2]
```diff
@@ -26,5 +26,9 @@
   "update_max_state_tree_size_per_byte2": {
     "enabled_block": 56300,
     "has_bandage": true
+  },
+  "update_bandwidth_budget_params": {
+    "enabled_block": -1,
```
Comment (platfowner) [inline blockchain-configs/testnet-prod/timer_flags.json:31]: ditto

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks for handling this!

File: blockchain-configs/mainnet-prod/timer_flags.json (hunk 1 above)
Comment (liayoo) [reply blockchain-configs/mainnet-prod/timer_flags.json]: done!

File: blockchain-configs/testnet-prod/timer_flags.json (hunk 2 above)
Comment (liayoo) [reply blockchain-configs/testnet-prod/timer_flags.json]: ditto

**Comment by liayoo** [comment]
Thanks for the fast review. Now merging~

**Pull request merged**